### PR TITLE
contributing: clarify scope

### PR DESCRIPTION
**Pull request by ikonst**

### CODE REVIEW

File: docs/contributing.rst [hunk 1]
```diff
@@ -61,6 +61,34 @@ Outside of data compatibility, follow the usual semver rules for API changes and
 to a major release.
 
 
+The Scope of the Library
+------------------------
+
+The purpose of this library is to provide a Pythonic ORM layer on top of DynamoDB to be used
```
Comment (jpinner-lyft) [inline docs/contributing.rst:67]: ODM not ORM

Comment (ikonst) [reply docs/contributing.rst:67]: agree, we just say "ORM-like" in the README so I thought...

Comment (jpinner-lyft) [reply docs/contributing.rst:67]: can we just change the readme as well -- we don't support any relationship management in this library

Comment (jpinner-lyft) [reply docs/contributing.rst:67]: or just drop the bullet point from the features list entirely

**Review by garrettheel — APPROVED** [review]

**Review by jpinner-lyft — APPROVED** [review]

**Pull request merged**